calculator.js

```diff
@@ -1,28 +1,134 @@
-
 const exampleAdditionInput = {
   num1: 3,
   num2: 5,
   operation: 'add',
 }
 
+// const badInput = {
+//   num1: 'dog',
+//   num2: 5,
+//   operation: '*',
+// }
+
 const calculateUserInput = function (error, promptInput) {
-  console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
+  // console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
+
+  // Check for invalid data
+
+  if (isNaN(promptInput.num1)) {
+    console.log(`${promptInput.num1} is not a number.`)
+    return
+  }
+  
+  if (isNaN(promptInput.num2)) {
+    console.log(`${promptInput.num2} is not a number.`)
+    return
+  }
```

Comment:
This totally works! But we could eliminate these two conditionals by adding  a `default` case to the switch case below. Check down there for another note!